packages/components/src/InputFile/InputFile.tsx

```diff
@@ -166,10 +179,13 @@ export function InputFile({
   onUploadStart,
   onUploadProgress,
   onUploadComplete,
+  onUploadError,
+  validator,
 }: InputFileProps) {
   const options: DropzoneOptions = {
     multiple: allowMultiple,
-    onDrop: useCallback(handleDrop, [uploadFile]),
```

Comment:
this `uploadFile` dependency was a recent addition from another PR. was this removal intentional?

Reply:
No that was unintentional. I missed this change during a merge